### PR TITLE
Login Component.

### PR DESCRIPTION
**Pull request by Deepakyadav19**

### CODE REVIEW

File: web/src/app/login/login.component.html [hunk 1]
```diff
@@ -1 +1,21 @@
...
+  <div class="container">
+    <div class="row">
+      <div class="col-md-4 login-sec">
+        <h2 class="text-center">Login Now</h2>
+        <img src="../../assets/outlooklogo.png" class="logo-design img-center img-fluid">
+      </div>
+      <div class="col-md-8 banner-sec">
+        <div class="carousel-item active">
+          <img class="d-block img-fluid" src="https://static.pexels.com/photos/33972/pexels-photo.jpg"
+            alt="First slide">
```
Comment (sonalichawla16) [inline web/src/app/login/login.component.html:11]: Add this image to assets. Dont use any asset directly from URL

File: web/src/app/login/login.component.html [hunk 2]
```diff
@@ -1 +1,21 @@
...
+          <div class="carousel-caption d-none d-md-block">
+            <div class="banner-text">
+              <h2>This is Heaven</h2>
+              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore
+                et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation</p>
+            </div>
```
Comment (sonalichawla16) [inline web/src/app/login/login.component.html:17]: Remove sample code.

File: web/src/app/login/login.component.scss [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  background: #F2EDF3;
+  float: left;
+  width: 100vw;
+  height: 100vh;
+  padding: 50px 0;
+}
+
+.banner-sec {
+  background: url(https://static.pexels.com/photos/33972/pexels-photo.jpg) no-repeat left bottom;
+  background-size: cover;
```
Comment (sonalichawla16) [inline web/src/app/login/login.component.scss:11]: Don't use any asset directly from URL

Comment (Deepakyadav19) [reply web/src/app/login/login.component.scss:11]: Now images are moved into assets folder

**Pull request merged**